src/gui/MixerView.cpp

```diff
@@ -169,17 +169,11 @@ MixerView::MixerView(Mixer* mixer) :
 	// adjust window size
 	layout()->invalidate();
 	resize(sizeHint());
-	if (parentWidget())
-	{
-		parentWidget()->resize(parentWidget()->sizeHint());
-	}
 	setFixedHeight(height());
+	layout()->setSizeConstraint(QLayout::SetMinimumSize);
 
 	// add ourself to workspace
 	QMdiSubWindow* subWin = mainWindow->addWindowedWidget(this);
```

Comment:
This is causing a build failure on Linux and MinGW because `subWin` is no longer used.
You could either remove `subWin` or use `(void)subWin;` so the compiler doesn't fail due to the unused variable.

Reply:
I opted for `[[maybe_unused]]` from c++17 since that's more descriptive